tools/agent-ctl/src/utils.rs

```diff
@@ -328,6 +337,338 @@ fn process_oci_to_grpc(p: &ociProcess) -> grpcProcess {
     }
 }
 
+fn idmaps_oci_to_grpc(res: &[oci::LinuxIdMapping]) -> protobuf::RepeatedField<grpcLinuxIDMapping> {
```

Comment:
The protocol between kata runtime and agent is in ttrpc. So it is a bit wrong in the function naming. Ditto the following functions.

I noted that the original names are wrong as well. Please do not follow them ;) Let's fix them instead.

Reply:
I fix these names, please help to check again. :-)